src-docs/src/views/combo_box/combo_box_example.js

```diff
@@ -245,31 +245,36 @@ const labelledbySnippet = `<EuiComboBox
 export const ComboBoxExample = {
   title: 'Combo box',
   intro: (
-    <>
-      <EuiText>
-        <p>
-          Use a <strong>EuiComboBox</strong> when the input has so many options
-          that the user needs to be able to search them, the user needs to be
-          able to select multiple options, and/or the user should have the
-          ability to specify a custom value in addition to selecting from a
-          predetermined list.
-        </p>
-        <EuiCallOut
-          iconType="accessibility"
-          title={
-            <>
-              You must add an accessible label to each instance of{' '}
-              <strong>EuiComboBox</strong>
-            </>
-          }
+    <EuiText>
+      <p>
+        Use a <strong>EuiComboBox</strong> when the input has so many options
+        that the user needs to be able to search them, the user needs to be able
+        to select multiple options, and/or the user should have the ability to
+        specify a custom value in addition to selecting from a predetermined
+        list. See{' '}
```

Comment:
This feels like a bit of a non-sequitur in relation to the rest of the preceding text - let's smooth over the transition (or consider making it another paragraph?)
```suggestion
        list. If you're unsure of which selection component to use, see{' '}
```